src/devices/maverick_et73x.c

```diff
@@ -57,6 +59,11 @@ static int maverick_et73x_callback(r_device *decoder, bitbuffer_t *bitbuffer)
     // decode the inner manchester encoding
     bitbuffer_manchester_decode(bitbuffer, 0, 0, &mc, 104);
 
+    // we require 7 bytes 13 nibble rounded up (  b[6] highest referance below )
+    if (mc.bits_per_row[0] < 56) {
```

Comment:
I'm not sure why, but all maverick tests now fail (rtl_433_tests). So I'd guess this is incorrect?